docs/versionhistory.rst

```diff
@@ -5,6 +5,12 @@ This library adheres to `Semantic Versioning 2.0 <http://semver.org/>`_.
 
 **UNRELEASED**
 
+- Improved compatibility with asyncio's eager task factories:
+
+  * Updated the annotation of ``TaskInfo.coro`` to allow it to be ``None``
```

Comment:
Is it possible for TaskInfo.coro to be None? Can't you only access it from a running task?

Reply:
This stemmed from a `mypy` complaint, but I think you're right – `TaskInfo` objects should never be created from completed tasks under any circumstances.